commerce/domain/tax/cart/calculator/classes/TaxCartCalculatorSample.cls

```diff
@@ -365,10 +374,10 @@ public class TaxCartCalculatorSample extends CartExtension.TaxCartCalculator {
             Double tierAdjustmentTax = (tierAdjustment!=null ? tierAdjustment : 0.00) * multiplier;
 
             CartExtension.CartItemPriceAdjustmentList itemizedPromotions = cartItem.getCartItemPriceAdjustments();
-
+          Iterator<CartExtension.CartItemPriceAdjustment> itemizedPromotionsIterator = itemizedPromotions.iterator();
```

Comment:
Nit: Add more spaces before iterator for alignment with other statements.

Reply:
done